product_security/recommendations_for_customers/secure_container_lifecycle_management.adoc

```diff
@@ -0,0 +1,92 @@
+are run with user, without a fixed UID, and not root privileges on the host.
+
+[[dedicated-container-content]]
+== Container Content
+
+Some key questions to consider when it comes to content security for containers
+include:
+
+* Will the contents of the containers compromise my infrastructure?
+* Are there known vulnerabilities in the application layer?
+* Are the runtime and OS layers up to date?
+* How frequently will the container be updated and how will you know when it s
+updated?
+
+Red Hat delivers trusted content via the Red Hat Container Catalog. However,
+there will be times when you need content that Red Hat does not provide. It is
+recommended that you use container scanning tools that use continuously updated
+vulnerability databases to be sure that you always have the latest information on
+known vulnerabilities when using container images from other sources.
+
+[[dedicated-container-registries]]
+== Container Registries
+
+{product-title} includes a private registry that can be used to manage your
+container images. The OpenShift registry provides role-based access controls
+that allow you to manage who can pull and push specific container images.
+{product-title} also supports pushing and pulling from other private registries
+you might already be using.
+
+[[dedicated-building-containers]]
+== Building Containers
+
+In a containerized environment, the software build process is the stage in the
+life cycle where application code is integrated with needed runtime libraries.
+Managing this build process is key to securing the software stack.
+{product-title} provides a number of capabilities for build management and
+security:
+
+* Source-to-image (S2I) is an open source framework for combining source
+code and base images. S2I makes it easy for your development and operations
+teams to collaborate on a reproducible build environment.
+
+* {product-title} includes an integrated instance of Jenkins for CI.
+{product-title} also includes rich RESTful APIs that you can use to integrate
+your own build or CI tools or private image registry, such as JFrog’s
+Artifactory.
+
+* A best practice for application security is to integrate automated security
+testing into your build or CI process. Static Application Security Testing
+(SAST) and Dynamic Applications Security Testing (DAST) tools like HP Fortify
+and IBM AppScan can be used with OpenShift environment along with Scanners for
+real-time checking against known vulnerabilities like Black Duck Hub and JFrog
+Xray.
+
+[[dedicated-deploying-containers]]
+== Deploying Containers
+
+In case anything falls through during the build process, or for situations where
+a vulnerability is discovered after an image has been deployed, you want yet
+another layer of security: tools for automated, policy-based deployment. Strong
+security includes automated policies that you can use to manage container
+deployment from a security point of view.
```

Comment:
At the end of the whole page, maybe we can add this reference link https://www.redhat.com/en/resources/container-security-openshift-cloud-devops-whitepaper#

Reply:
Good idea. I am adding an Additional Resources section with this link